threedi_scenario_downloader/downloader.py

```diff
@@ -882,6 +939,31 @@ def get_raster_from_json(scenario_json, raster_code, subendpoint=None):
     return raster
 
 
+def time_in_range(start, end, x):
+    """Return true if x is in the range [start, end]"""
+    if start <= end:
+        return start <= x <= end
+    else:
+        return start <= x or x <= end
```

Comment:
Deze klopt niet. Tenminste, de hele if/else is niet nodig denk ik.
Komt het voor dat de startdatum *na* de einddatum ligt?
En zo ja, dan krijgt x het zeker niet voor elkaar om na start èn voor het eind te liggen :-)

Reply:
Deze klopt nog steeds niet, hoor. Ik ga er van uit dat start voor eind ligt en dat de fout dus niet op zal treden. Maar...

```
start = 10
end = 0
# end < start, raar genoeg? Maar goed, spul moet nog steeds tussen 0 en 10 liggen.
x = -1  # ligt er niet tussen
10 <= -1 or -1 <= 0  # Dit is waar, want -1 is kleiner dan 0
```

Dus... TODO als het echt voorkomt.